userinterface/ViewModels/Mapping/MappingViewModel.cs

```diff
@@ -14,15 +14,13 @@ public MappingViewModel(BE.MappingModel mappingBE, BE.MappingsModel mappingsBE)
         }
 
         public BE.MappingModel MappingBE { get; }
```

Comment:
Can we keep the newline between properties? To me, properties are meant to be encapsulation of getting\setting local members, and newlines demonstrate higher-level sections such as encapsulations more clearly.